lib/src/runner/browser/browser_manager.dart

```diff
@@ -272,7 +272,7 @@ class BrowserManager {
   }
 
   /// The callback for handling messages received from the host page.
-  void _onMessage(Map message) {
```

Comment:
I don't understand this. Is it possible to fix the place where this is used rather than loosen the argument?

Reply:
I'd prefer this as well. I'd like to avoid dynamic calls, even though in practice `--no-implicit-dynamic` isn't really in shape to be enabled yet.

Reply:
> Is it possible to fix the place where this is used rather than loosen the argument?

Done, though it feels a little silly in this case. `_onMessage` is only called by that one place and `Map` isn't super precise either.